src/libraries/System.Security.Cryptography/src/System/Security/Cryptography/PemEncoding.cs

```diff
@@ -62,37 +89,74 @@ public static PemFields Find(ReadOnlySpan<char> pemData)
         /// <c>true</c> if PEM-encoded data was found; otherwise <c>false</c>.
         /// </returns>
         /// <remarks>
-        /// IETF RFC 7468 permits different decoding rules. This method
-        /// always uses lax rules.
+        ///   <para>IETF RFC 7468 permits different decoding rules. This method always uses lax rules.</para>
+        ///   <para>
+        ///     This does not validate the UTF-8 data outside of encapsulation boundaries and is ignored. It is the caller's
+        ///     responsibility to ensure the entire input is UTF-8 if required.
```

Comment:
Looks like the doc comment ended up on the UTF-16 version, instead of the UTF-8 version